[2024-05-26 21:24:16,680] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [queued]>
[2024-05-26 21:24:16,691] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [queued]>
[2024-05-26 21:24:16,693] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-26 21:24:16,694] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-05-26 21:24:16,694] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-26 21:24:16,712] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): save_into_db> on 2024-05-25T02:00:00+00:00
[2024-05-26 21:24:16,719] {standard_task_runner.py:52} INFO - Started process 178 to run task
[2024-05-26 21:24:16,724] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'load_init_data', 'save_into_db', '2024-05-25T02:00:00+00:00', '--job-id', '402', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/load_init_data.py', '--cfg-path', '/tmp/tmpizik168i', '--error-file', '/tmp/tmp8fin517h']
[2024-05-26 21:24:16,727] {standard_task_runner.py:77} INFO - Job 402: Subtask save_into_db
[2024-05-26 21:24:16,780] {logging_mixin.py:104} INFO - Running <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [running]> on host ff5273ee61d8
[2024-05-26 21:24:16,833] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=load_init_data
AIRFLOW_CTX_TASK_ID=save_into_db
AIRFLOW_CTX_EXECUTION_DATE=2024-05-25T02:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-25T02:00:00+00:00
[2024-05-26 21:24:17,270] {python.py:118} INFO - Done. Returned value was: None
[2024-05-26 21:24:17,288] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=load_init_data, task_id=save_into_db, execution_date=20240525T020000, start_date=20240526T212416, end_date=20240526T212417
[2024-05-26 21:24:17,336] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-26 21:24:17,377] {local_task_job.py:146} INFO - Task exited with return code 0
[2024-05-27 21:19:34,537] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [queued]>
[2024-05-27 21:19:34,547] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [queued]>
[2024-05-27 21:19:34,547] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-27 21:19:34,548] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-05-27 21:19:34,548] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-27 21:19:34,553] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): save_into_db> on 2024-05-25T02:00:00+00:00
[2024-05-27 21:19:34,556] {standard_task_runner.py:52} INFO - Started process 195 to run task
[2024-05-27 21:19:34,560] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'load_init_data', 'save_into_db', '2024-05-25T02:00:00+00:00', '--job-id', '503', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/load_init_data.py', '--cfg-path', '/tmp/tmpo8ae58lg', '--error-file', '/tmp/tmp_v0q4fdv']
[2024-05-27 21:19:34,562] {standard_task_runner.py:77} INFO - Job 503: Subtask save_into_db
[2024-05-27 21:19:34,591] {logging_mixin.py:104} INFO - Running <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [running]> on host 1f334d963743
[2024-05-27 21:19:34,622] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=load_init_data
AIRFLOW_CTX_TASK_ID=save_into_db
AIRFLOW_CTX_EXECUTION_DATE=2024-05-25T02:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-25T02:00:00+00:00
[2024-05-27 21:19:34,818] {python.py:118} INFO - Done. Returned value was: None
[2024-05-27 21:19:34,826] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=load_init_data, task_id=save_into_db, execution_date=20240525T020000, start_date=20240527T211934, end_date=20240527T211934
[2024-05-27 21:19:34,846] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-27 21:19:34,864] {local_task_job.py:146} INFO - Task exited with return code 0
[2024-06-09 14:02:28,995] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [queued]>
[2024-06-09 14:02:29,123] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [queued]>
[2024-06-09 14:02:29,129] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-09 14:02:29,137] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-09 14:02:29,146] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-09 14:02:29,202] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): save_into_db> on 2024-05-25T02:00:00+00:00
[2024-06-09 14:02:29,227] {standard_task_runner.py:52} INFO - Started process 268 to run task
[2024-06-09 14:02:29,230] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'load_init_data', 'save_into_db', '2024-05-25T02:00:00+00:00', '--job-id', '41', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/load_init_data.py', '--cfg-path', '/tmp/tmp92f1q2sr', '--error-file', '/tmp/tmpz6lisef2']
[2024-06-09 14:02:29,243] {standard_task_runner.py:77} INFO - Job 41: Subtask save_into_db
[2024-06-09 14:02:29,380] {logging_mixin.py:104} INFO - Running <TaskInstance: load_init_data.save_into_db 2024-05-25T02:00:00+00:00 [running]> on host 97b873bf6d4d
[2024-06-09 14:02:29,508] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=load_init_data
AIRFLOW_CTX_TASK_ID=save_into_db
AIRFLOW_CTX_EXECUTION_DATE=2024-05-25T02:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-25T02:00:00+00:00
[2024-06-09 14:02:31,267] {python.py:118} INFO - Done. Returned value was: None
[2024-06-09 14:02:31,323] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=load_init_data, task_id=save_into_db, execution_date=20240525T020000, start_date=20240609T140229, end_date=20240609T140231
[2024-06-09 14:02:31,522] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-09 14:02:31,572] {local_task_job.py:146} INFO - Task exited with return code 0
